src/Cosmonaut/CosmosStoreSettings.cs

```diff
@@ -52,18 +59,18 @@ public class CosmosStoreSettings
             ConnectionPolicy connectionPolicy = null,
             IndexingPolicy indexingPolicy = null,
             int defaultCollectionThroughput = CosmosConstants.MinimumCosmosThroughput)
-            : this(databaseName, 
-                  new Uri(endpointUrl), 
+            : this(databaseName,
+                  new Uri(endpointUrl),
                   authKey,
                   connectionPolicy,
                   indexingPolicy,
                   defaultCollectionThroughput)
         {
         }
-        
+
         public CosmosStoreSettings(
```

Comment:
`Interceptors = new List<IQueryInterceptor>()` is missing from this constructor

Reply:
Resolved